### DISCUSSION TITLE
shadow property

### DISCUSSION THREAD

**Discussion by vikas-growisto** [fastn-stack/fastn#585] [Q&A] [answered]
```
-- ftd.text: FifthTry
shadow-offset-x: 14
shadow-offset-y: 10
shadow-blur: 5
shadow-size: 4
shadow-color: yellow
```
The properties are not working can you provide an example that is working with the properties.

**Comment by Arpita-Jaiswal** [comment]
The shadow properties are not yet supported. Please subscribe to https://github.com/ftd-lang/ftd/issues/591 to get updates and notifications.

**Comment by amitu** [comment, marked as answer]
[Shadow is now supported](https://github.com/fastn-stack/fastn/commit/a91748d959e49595f60751a879d5312f6cf2d460#diff-2a9ec7380484a61c619ec837a6377caaf3195b42b0fa3c9f9bd1ed2cb5d2f935R18):

You can create shadow variable like this:

```ftd
-- ftd.shadow some-shadow:
y-offset.px: -5
x-offset.px: 10
spread.px: 3
blur.px: 1
color: red
```

And then you can use it on any element:

```ftd
-- ftd.text: hello
shadow: $some-shadow
```
